drivers/wifi/nrf700x/osal/hw_if/hal/inc/hal_structs.h

```diff
@@ -216,7 +216,7 @@ struct wifi_nrf_hal_dev_ctx {
 	unsigned long addr_rpu_pktram_base_tx;
 	unsigned long addr_rpu_pktram_base_rx;
 	unsigned long addr_rpu_pktram_base_rx_pool[MAX_NUM_OF_RX_QUEUES];
-
+	unsigned long last_buf_addr;
```

Comment:
Call this as `addr_last_buf` to maintain consistency with other variable names above